<launch>
    <group ns="robot_0">
    	<param name="tf_prefix" value="robot_0" />
	    <node pkg="tf" type="static_transform_publisher" name="link_broadcaster" args="6.5 18.5 0 0 0 0 /map /robot_0/odom 100" />
        <node pkg="formation" type="robot.py" name="robot" args="4" output="screen" />
    </group>
</launch>

